From 56466328a5515f28e2fb9d78e155dadeab38b9c2 Mon Sep 17 00:00:00 2001
From: Henrik Grimler <henrik@grimler.se>
Date: Sun, 16 Jan 2022 16:13:44 +0100
Subject: [PATCH 15/16] dt-bindings: arm: samsung: document Klimt LTE board
 binding

Add binding for Galaxy Tab S 8.4" LTE variant, based on Exynos 5420
with codename klimt-lte. The tablet has similar hardware to Klimt
WiFi but with:

* an Intel 6360 modem
* a sim-card reader
* a TMD3782 light sensor, instead of Capella CM3323
* a Semtech SX9500 proximity sensor

Signed-off-by: Henrik Grimler <henrik@grimler.se>
---
 .../devicetree/bindings/arm/samsung/samsung-boards.yaml          | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml b/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml
index abac01def63c..83142ab8526d 100644
--- a/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml
+++ b/Documentation/devicetree/bindings/arm/samsung/samsung-boards.yaml
@@ -142,6 +142,7 @@ properties:
               - insignal,arndale-octa           # Insignal Arndale Octa
               - samsung,chagall-wifi            # Samsung SM-T800
               - samsung,klimt-wifi              # Samsung SM-T700
+              - samsung,klimt-lte               # Samsung SM-T705
               - samsung,smdk5420                # Samsung SMDK5420 eval
           - const: samsung,exynos5420
           - const: samsung,exynos5
-- 
2.34.1

